src/parse_args.ts

```diff
@@ -23,16 +23,14 @@
  *
  */
 
-import { program } from 'commander';
+import { program, Option } from 'commander';
 import { CliArgs } from './common_types';
 import { reporters } from './reporters';
 
-const availableReporters = Object.keys(reporters)
-  .map(r => String(r))
-  .join();
-
 /* eslint-disable-next-line @typescript-eslint/no-var-requires */
 const { name, version } = require('../package.json');
+const allowedCapabilities = ['trace', 'filmstrips', 'metrics'];
```

Comment:
I'm not sure it makes sense to have a strict list of capabilities. If we do that we may as well just have the individual flags which are easier to understand.

The point of capabilities was that these were capabilities you could ask for but if not present would not cause an error.

This is less relevant given our plan to have the `--rich-events` flag, which solves the same problem from heartbeat'- perspective. However, I still think the capabilities flag is a nice thing to have so long as we reserve it for things that are not strict requirements. We could consider adding an info or warning message for missing capabilities

Reply:
++, My mistake and feature flags shouldn't be a restrictive list. 

> We could consider adding an info or warning message for missing capabilities

I like that idea. Let me incorporate your feedback.